Makefile

```diff
@@ -138,6 +140,10 @@ install-tools:
 otelcol:
 	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcol_$(GOOS)_$(GOARCH) $(BUILD_INFO) ./cmd/otelcol
 
+.PHONY: run
+run:
+	GO111MODULE=on go run --race ./cmd/otelcol/... --config ${RUN_CONFIG}
```

Comment:
Opportunistic update to bring the run make target from contrib repo here. It's very useful when iterating fast on a feature.